docs/reference/dataQuality.md

```diff
@@ -57,3 +59,19 @@ dataObjects {
     }]
 }
 ```
+
+## Expectations on Actions
+Its also possible to define certain expectations on the Action. They are used to measure and validate metrics about the transformation process, e.g. transfer rate (output/input count) and completness (total output count/total input count).
+
+```
+dataObjects {
+  testAction {
+    ...
+    expectations = [{
+      type = CompletnessExpectation
```

Comment:
I see there's also a typo in the class itself. Will create a separate issue for that.

Reply:
@zzeekk See #880